app/components/ui/files/bread-crumbs/component.js

```diff
@@ -1,12 +1,27 @@
 import Component from '@ember/component';
 import { inject as service } from '@ember/service';
+import { computed } from '@ember/object';
 import layout from './template';
 
+const taleStatus = Object.create({
```

Comment:
Can we rename that to `taleAccess` or just something other than `taleStatus`. I'm concerned it's gonna be confusing with an actual TaleStatus (`{PREPARING, READY, ERROR}`)

Reply:
:+1: I just copy-pasted this from another place where we use the same pattern and am less concerned with the naming conventions of legacy code

That said, I went ahead updated the enum name to `TaleAccessLevel` (in this usage only)